pkg/fedramp/config.go

```diff
@@ -83,11 +88,11 @@ var AdminTokenURLs = map[string]string{
 	"integration": fmt.Sprintf("https://rh-ocm-appsre-integration.%s", cognitoURL),
 }
 
-// ClientIDs allows the value of the `--env` option to map to the various AWS Cognito user pool clients.
+// ClientIDs allows the value of the `--env` option to map to the Keycloak clients.
 var ClientIDs = map[string]string{
-	"production":  "72ekjh5laouap6qcfis521jlgi",
-	"staging":     "1lb687dlpsmsfuj53r3je06vpp",
-	"integration": "20fbrpgl28f8oehp6709mk3nnr",
+	"production":  "console-dot",
+	"staging":     "console-dot",
```

Comment:
Since the stg/prod token URLs are not changing _yet_, I would keep these commented out/removed until we migrated all envs.